inference-engine/tests/functional/plugin/myriad/shared_tests_instances/skip_tests_config.cpp

```diff
@@ -23,5 +23,7 @@ std::vector<std::string> disabledTestPatterns() {
         R"(.*(IEClassLoadNetwork).*(QueryNetworkMULTIWithHETERONoThrow_V10|QueryNetworkHETEROWithMULTINoThrow_V10).*)",
         // TODO: Issue: 34348
         R"(.*IEClassGetAvailableDevices.*)",
+        // TODO: Issue: 38643
+        R"(.*EltwiseLogicalNotInt.*)",
```

Comment:
Do you plan to fix this issue about logical-not?

Reply:
Not me, but it is planned to be fixed for the next release.

Reply:
Do you mean 2021.2 by next release?

Reply:
Yes.